### ISSUE TITLE
[BUG] Eslint not detecting tailwind.config.js

### ISSUE THREAD

**Issue by AlexC01** [francoismassart/eslint-plugin-tailwindcss#170]
**Describe the bug**
Im having problems setting up tailwind on my React project. I am using eslint, and I wanted to add tailwind and asked me to add it as a project, but then it says it is not correct

**To Reproduce**
The error is the following: 
![image](https://user-images.githubusercontent.com/49468322/192420246-d7cdab64-59a5-48f6-aa97-aeca4a74d45d.png)

This is the Eslint configuration:
![image](https://user-images.githubusercontent.com/49468322/192420878-57be4c8e-4e84-43fe-8e4c-d1c4eda2f2a2.png)
**Expected behavior**
It should detect it as the same path

**Environment (please complete the following information):**
 - OS Windows 10
 - Softwares + version used:
     -  VSCode 1.71.2
     - npm 8.11.0
     - node 16.15.1
     - tailwindcss ^3.1.8
     - eslint-plugin-tailwindcss ^3.6.2

**eslint config file or live demo**
Repo to the eslint: https://github.com/AlexC01/cat-wiki/blob/master/frontend/.eslintrc.json

**Comment by EwenDC** [comment]
You're not defining the Tailwind config file correctly. `eslint-plugin-tailwindcss` is not a parser and doesn't read the parserOptions. parserOptions.project should only be used for defining the location of your `tsconfig.json` file. You need to define the location of the Tailwind config using the ESLint plugin settings, like so:
```json
{
  "env": {
    "browser": true,
    "es2021": true
  },
  "extends": [
    "plugin:react/recommended",
    "standard-with-typescript",
    "plugin:jsx-a11y/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:react-hooks/recommended",
    "plugin:tailwindcss/recommended",
    "plugin:import/typescript",
    "prettier"
  ],
  "parser": "@typescript-eslint/parser",
  "overrides": [],
  "parserOptions": {
    "ecmaFeatures": { "jsx": true },
    "ecmaVersion": "latest",
    "project": "frontend/tsconfig.json",
    "sourceType": "module"
  },
  "settings": {
    "tailwindcss": {
      "config": "frontend/tailwind.config.js"
    }
  },
  "plugins": ["react", "@typescript-eslint", "react-hooks", "prettier", "tailwindcss"],
  "rules": {
    "prettier/prettier": "error",
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx", ".tsx"] }],
    "@typescript-eslint/explicit-function-return-type": "off"
  }
}
```

If the reason you tried adding `frontend/tailwind.config.js` to parserOptions.project was because of the following error:
```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: frontend/tailwind.config.js.
The file must be included in at least one of the projects provided.eslint
```
That's because ESLint is attempting to parse the file using `typescript-eslint`, but within your `tsconfig.json` you excluded that file (either explicitly using the "exclude" setting, or you used "include" to define paths that don't include the path for `frontend/tailwind.config.js`). You need to either tell ESLint to fall back to a default parser to parse files that are excluded in `tsconfig.json`, or tell ESLint to skip linting those files all together with a `.eslintignore` file. [More information here.](https://stackoverflow.com/questions/58510287/parseroptions-project-has-been-set-for-typescript-eslint-parser)

**Comment by francoismassart** [comment]
Looks like we can close this one. Re-open it with a exhaustive comment if needed @AlexC01
